src/main/java/org/levelup/labradoodle/models/entities/kladr/City.java

```diff
@@ -21,6 +23,9 @@
     @OneToMany(mappedBy = "city_id")
     private Collection<Street> street= new ArrayList<Street>();
 
+    @OneToMany(mappedBy = "city")
+    private Collection<Restaurant> restaurants=new ArrayList<>();
```

Comment:
private Collection<Restaurant> restaurants  Not collection..
private List<Restaurant> restaurants